### PR TITLE
[zh] Sync #14542 with typo fix into Chinese

### PR DESCRIPTION
**Pull request by wilsonwu**
## Description

Sync #14542 with typo fix into Chinese

## Reviewers

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [x] Localization/Translation

### CODE REVIEW

File: content/zh/docs/setup/upgrade/in-place/index.md [hunk 1]
```diff
@@ -96,14 +94,12 @@ Istio 的升级指令同样可以执行回退操作。
 * 安装的 Istio 版本与回退的目标版本之间至最多差一个 minor 版本。例如，您可以从 1.7.x
```
Comment (hanxiaop) [inline content/zh/docs/setup/upgrade/in-place/index.md:94]: Should this word `minor` be translated?

Comment (wilsonwu) [reply content/zh/docs/setup/upgrade/in-place/index.md:94]: I think yes.

Comment (wilsonwu) [reply content/zh/docs/setup/upgrade/in-place/index.md:94]: Done

Comment (windsonsea) [reply content/zh/docs/setup/upgrade/in-place/index.md:94]: 小版本、次要版本 都可以吧

**Review by hanxiaop — APPROVED** [review]
Looks good, thanks

**Review by windsonsea — APPROVED** [review]
thanks

**Pull request merged**